cmd/new_test.go

```diff
@@ -140,7 +142,8 @@ func TestCreatingStackWithArgsSpecifiedFullNameSucceeds(t *testing.T) {
 	err := runNew(args)
 	assert.NoError(t, err)
 
-	assert.Equal(t, fullStackName, loadStackName(t))
+	assert.Equal(t, stackName, loadStackName(t))
```

Comment:
Why did we expect to get the full name back and now we expect to get the short name?

Reply:
Good question. It appears that when the stack already exists, we bring in the fully-qualified name as the stack name. However, when we create a new stack, it only has the final part of the name. We should probably resolve this inconsistency, so I'll do that as part of this PR as well instead of leaving it like this.

Reply:
Digging further, the reason this used to use fullStackName is because of pkg/backend/httpstate/stack.go:58 -- which matches up. However, previously, this test brought in an existing stack (due to statefulness), which causes the full stack reference to be the name. Without knowing if anything depends on this behavior, I'm going to leave it as is. Does that sound good?